server/src/server.ts

```diff
@@ -0,0 +1,79 @@
+import express from "express";
+import mongoose from "mongoose";
+import dotenv from "dotenv";
+import cors from "cors";
+
+import { exerciseModel } from "./models/productModel";
+
+dotenv.config();
+const app = express();
+const MONGO_URI = `mongodb+srv://root:${process.env.DBPASS}@gymscheduler.c8rayg3.mongodb.net/`;
+
+app.use(
+  express.json(),
+  cors({
+    origin: process.env.FRONTEND_ADDRESS,
+  })
+);
+
+app.post("/exerciseDay", async (req, res) => {
+  try {
+    const exercise = await exerciseModel.create(req.body);
+    res.status(200).json(exercise);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({ message: error });
+  }
+});
+
+app.get("/exerciseDay", async (req, res) => {
+  try {
+    const exercises = await exerciseModel.find({});
```

Comment:
The application does not have users, is this correct? It returns all exercises from database

Reply:
For now i want to leave it as it is, because users will be added as a future feature